modules/contact-form/grunion-contact-form.php

```diff
@@ -1995,14 +1995,16 @@ function process_submission() {
 		 * @param string|array $to Array of valid email addresses, or single email address.
 		 */
 		$to = (array) apply_filters( 'contact_form_to', $to );
+		$reply_to_addr = $to[0]; // get just the address part before the name part is added
+
 		foreach ( $to as $to_key => $to_value ) {
 			$to[ $to_key ] = Grunion_Contact_Form_Plugin::strip_tags( $to_value );
+			$to[ $to_key ] = self::add_name_to_address( $to_value );
```

Comment:
Would it be worth attempting to see if there is a "Name" field, and if so, use that instead of the email address for the name portion?